src/pages/cb-bank-accounts/forms/CbBankAccountsForm.js

```diff
@@ -0,0 +1,239 @@
+                required
+                maxAccess={maxAccess}
+                onChange={formik.handleChange}
+                onClear={() => formik.setFieldValue('name', '')}
+                error={formik.touched.name && Boolean(formik.errors.name)}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <CustomNumberField
+                name='accountNo'
+                label={labels.accountNo}
+                value={formik.values.accountNo}
+                required
+                maxAccess={maxAccess}
+                onChange={formik.handleChange}
+                onClear={() => formik.setFieldValue('accountNo', '')}
+                error={formik.touched.accountNo && Boolean(formik.errors.accountNo)}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <ResourceComboBox
+                endpointId={SystemRepository.Currency.qry}
+                name='currencyId'
+                label={labels.currencyName}
+                valueField='recordId'
+                displayField={['reference', 'name']}
+                columnsInDropDown={[
+                  { key: 'reference', value: 'Reference' },
+                  { key: 'name', value: 'Name' }
+                ]}
+                values={formik.values}
+                required
+                maxAccess={maxAccess}
+                onChange={(event, newValue) => {
+                  formik.setFieldValue('currencyId', newValue?.recordId || null)
+                }}
+                error={formik.touched.currencyId && Boolean(formik.errors.currencyId)}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <ResourceComboBox
+                datasetId={DataSets.ACTIVE_STATUS}
+                name='activeStatus'
+                label={labels.activeStatus}
+                valueField='key'
+                displayField='value'
+                values={formik.values}
+                required
+                maxAccess={maxAccess}
+                onChange={(event, newValue) => {
+                  formik.setFieldValue('activeStatus', newValue?.key)
+                }}
+                error={formik.touched.activeStatus && Boolean(formik.errors.activeStatus)}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <ResourceComboBox
+                endpointId={CashBankRepository.CbCashGroup.qry}
+                name='groupId'
+                label={labels.groupName}
+                valueField='recordId'
+                displayField={['reference', 'name']}
+                columnsInDropDown={[
+                  { key: 'reference', value: 'Reference' },
+                  { key: 'name', value: 'Name' }
+                ]}
+                values={formik.values}
+                maxAccess={maxAccess}
+                onChange={(event, newValue) => {
+                  if (newValue) {
+                    formik.setFieldValue('groupId', newValue?.recordId)
```

Comment:
better to use ternary operator 
formik.setFieldValue('groupId', newValue ? newValue.recordId : '');